app/app.js

```diff
@@ -76,7 +76,7 @@ app.use(require('express-session')({
     secret: 'denmarkisbetterthanswedenandfinland',
     resave: false,
     saveUninitialized: false,
-    cookie: { secure: true, httpOnly: true, sameSite: true}
+    cookie: { secure: false, httpOnly: true, sameSite: true} //Enable secure to true while on HTTPS
```

Comment:
That should probably be somehow exposed to config or as PROD environmental variable setting?

Reply:
It's easy to forget about it

Reply:
There's a lot of things that needs changed when we've implemented SSL, adding these all to the env would make quite a messy file. Or I could add the env variable of SSL (boolean) and determine the values / behaviour based off that, however that would need to be it's own PR. What do you think?

Reply:
No, I mean one PROD flag, that turns the all the security things ON (that are turned off as we develop it)

Reply:
SSL flag is also an option.  We shouldn't have to track all these comments

Reply:
I've changed it to secure: DEV_MODE ? false : true, instead of secure: !DEV_MODE for readability purposes.